<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<class>EditUserDialog</class>
<widget class="QDialog" name="EditUserDialog">
<property name="geometry">
<rect>
<x>0</x>
<y>0</y>
<width>846</width>
<height>486</height>
</rect>
</property>
<property name="windowTitle">
<string>Dialog</string>
</property>
<layout class="QGridLayout" name="gridLayout_2">
<item row="0" column="0">
<widget class="QGroupBox" name="personal_info_groupBox">
<property name="title">
<string>Personal Info</string>
</property>
<layout class="QGridLayout" name="gridLayout">
<item row="3" column="0">
<layout class="QHBoxLayout" name="horizontalLayout_4">
<item alignment="Qt::AlignLeft">
<widget class="QLabel" name="label_4">
<property name="text">
<string>Password:</string>
</property>
<property name="buddy">
<cstring>password_lineEdit_4</cstring>
</property>
</widget>
</item>
<item alignment="Qt::AlignRight">
<widget class="QLineEdit" name="password_lineEdit_4">
<property name="minimumSize">
<size>
<width>300</width>
<height>0</height>
</size>
</property>
<property name="inputMask">
<string/>
</property>
<property name="clearButtonEnabled">
<bool>true</bool>
</property>
</widget>
</item>
</layout>
</item>
<item row="1" column="0">
<layout class="QHBoxLayout" name="horizontalLayout">
<item alignment="Qt::AlignLeft">
<widget class="QLabel" name="label">
<property name="text">
<string>Login:</string>
</property>
</widget>
</item>
<item>
<spacer name="horizontalSpacer_2">
<property name="orientation">
<enum>Qt::Horizontal</enum>
</property>
<property name="sizeType">
<enum>QSizePolicy::Fixed</enum>
</property>
<property name="sizeHint" stdset="0">
<size>
<width>40</width>
<height>20</height>
</size>
</property>
</spacer>
</item>
<item>
<widget class="QLineEdit" name="login_lineEdit_2"/>
</item>
</layout>
</item>
<item row="0" column="0">
<layout class="QHBoxLayout" name="horizontalLayout_11">
<item>
<widget class="QLabel" name="label_10">
<property name="text">
<string>User Number:</string>
</property>
</widget>
</item>
<item>
<spacer name="horizontalSpacer">
<property name="orientation">
<enum>Qt::Horizontal</enum>
</property>
<property name="sizeHint" stdset="0">
<size>
<width>40</width>
<height>20</height>
</size>
</property>
</spacer>
</item>
<item>
<widget class="QLineEdit" name="id_lineEdit">
<property name="maximumSize">
<size>
<width>60</width>
<height>16777215</height>
</size>
</property>
<property name="readOnly">
<bool>true</bool>
</property>
</widget>
</item>
</layout>
</item>
<item row="2" column="0">
<layout class="QHBoxLayout" name="horizontalLayout_5">
<item alignment="Qt::AlignLeft">
<widget class="QLabel" name="label_5">
<property name="text">
<string>Email:</string>
</property>
<property name="buddy">
<cstring>password_lineEdit_4</cstring>
</property>
</widget>
</item>
<item alignment="Qt::AlignRight">
<widget class="QLineEdit" name="email_lineEdit_4">
<property name="minimumSize">
<size>
<width>300</width>
<height>0</height>
</size>
</property>
<property name="inputMask">
<string/>
</property>
<property name="clearButtonEnabled">
<bool>true</bool>
</property>
</widget>
</item>
</layout>
</item>
</layout>
</widget>
</item>
<item row="0" column="1">
<layout class="QVBoxLayout" name="verticalLayout">
<item>
<widget class="QPushButton" name="edit_pushButton">
<property name="text">
<string>Edit</string>
</property>
<property name="icon">
<iconset resource="resources.qrc">
<normaloff>:/images/images/pencil3.png</normaloff>:/images/images/pencil3.png</iconset>
</property>
</widget>
</item>
<item>
<widget class="QPushButton" name="cancel_pushButton_2">
<property name="text">
<string>Cancel</string>
</property>
<property name="icon">
<iconset resource="resources.qrc">
<normaloff>:/images/images/1413781093_Delete.png</normaloff>:/images/images/1413781093_Delete.png</iconset>
</property>
</widget>
</item>
<item>
<widget class="QPushButton" name="save_pushButton_3">
<property name="text">
<string>Save</string>
</property>
<property name="icon">
<iconset resource="resources.qrc">
<normaloff>:/images/images/document_save.png</normaloff>:/images/images/document_save.png</iconset>
</property>
</widget>
</item>
</layout>
</item>
<item row="1" column="0">
<widget class="QGroupBox" name="users_groupBox_3">
<property name="title">
<string>Users</string>
</property>
<layout class="QGridLayout" name="gridLayout_3">
<item row="0" column="0">
<layout class="QHBoxLayout" name="horizontalLayout_13">
<item alignment="Qt::AlignLeft">
<widget class="QLabel" name="label_11">
<property name="text">
<string>Search for</string>
</property>
</widget>
</item>
<item alignment="Qt::AlignLeft">
<widget class="QComboBox" name="comboBox">
<property name="minimumSize">
<size>
<width>150</width>
<height>0</height>
</size>
</property>
</widget>
</item>
<item alignment="Qt::AlignLeft">
<widget class="QDateEdit" name="search_dateEdit">
<property name="calendarPopup">
<bool>true</bool>
</property>
</widget>
</item>
<item>
<widget class="QLineEdit" name="search_lineEdit_2">
<property name="minimumSize">
<size>
<width>300</width>
<height>0</height>
</size>
</property>
</widget>
</item>
<item>
<spacer name="horizontalSpacer_3">
<property name="orientation">
<enum>Qt::Horizontal</enum>
</property>
<property name="sizeHint" stdset="0">
<size>
<width>40</width>
<height>20</height>
</size>
</property>
</spacer>
</item>
</layout>
</item>
<item row="1" column="0">
<widget class="QTableView" name="user_tableView"/>
</item>
</layout>
</widget>
</item>
<item row="2" column="0" colspan="2">
<widget class="QDialogButtonBox" name="buttonBox_2">
<property name="standardButtons">
<set>QDialogButtonBox::Ok</set>
</property>
</widget>
</item>
</layout>
</widget>
<resources>
<include location="resources.qrc"/>
</resources>
<connections>
<connection>
<sender>buttonBox_2</sender>
<signal>accepted()</signal>
<receiver>EditUserDialog</receiver>
<slot>accept()</slot>
<hints>
<hint type="sourcelabel">
<x>774</x>
<y>459</y>
</hint>
<hint type="destinationlabel">
<x>775</x>
<y>379</y>
</hint>
</hints>
</connection>
</connections>
</ui>
